vnext/ReactUWP/Modules/Animated/NativeAnimatedModule.cpp

```diff
@@ -39,6 +39,10 @@ const char *NativeAnimatedModule::s_addAnimatedEventToViewName{
     "addAnimatedEventToView"};
 const char *NativeAnimatedModule::s_removeAnimatedEventFromViewName{
     "removeAnimatedEventFromView"};
+const char *NativeAnimatedModule::s_startListeningToAnimatedNodeValueName{
+    "startListeningToAnimatedNodeValue" };
+const char *NativeAnimatedModule::s_stopListeningToAnimatedNodeValueName{
+    "stopListeningToAnimatedNodeValue" };
```

Comment:
please rebase/reformat before merging (since we now use 120 columns for formatting :) )